bin/helpers/archiver.js

```diff
@@ -36,10 +36,13 @@ const archiveSpecs = (runSettings, filePath) => {
 
     archive.pipe(output);
 
+    var packageJSON = JSON.stringify({devDependencies: runSettings.npm_dependencies}, null, 4);
```

Comment:
need to handle the case where the user is not setting any `npm_dependencies` or just removed the key manually.

Reply:
Done